Initialize the client at first connection for programs that have not done so.

--- a/lib/Rconnect.c
+++ b/lib/Rconnect.c
@@ -75,6 +75,10 @@
 		return connect(s, name, namelen);
 	}
 
+#if SOCKS_CLIENT /* may be called before normal init, log to right place */
+	clientinit();
+#endif     
+   
 	slog(LOG_DEBUG, "%s: %s",
 	function, sockaddr2string(name, namestring, sizeof(namestring)));
 
